lightkurve/utils.py

```diff
@@ -436,3 +438,19 @@ def plot_image(image, ax=None, scale='linear', origin='lower',
 class LightkurveWarning(Warning):
     """Class for all Lightkurve warnings."""
     pass
+
+
+@contextlib.contextmanager
+def suppress():
+    '''
```

Comment:
{nitpicking}
Per [PEP257](https://www.python.org/dev/peps/pep-0257/), a common Python convention is to use double quotes instead of single quotes for docstrings, and to put one-line docstring on one line, i.e.
```suggestion
    """A simple decorator to suppress function print outputs."""
```

Reply:
Good to know 👍